### PR TITLE
feat(web): implement comments count column in asset list table

### PR DESCRIPTION
**Pull request by nourbalaha**
# Overview
In this PR I implements the comments count column in asset list table

## What I have done
- I added thread to asset schema.
- I added Comment type to asset.type.ts.
- I added convertComment to convertAsset.ts.
- I implemented comments count in asset list table.

### CODE REVIEW

File: web/src/components/molecules/Asset/AssetListTable/index.tsx [hunk 1]
```diff
@@ -56,7 +56,10 @@ const AssetListTable: React.FC<AssetListTableProps> = ({
       title: () => <Icon icon="message" />,
       dataIndex: "commentsCount",
       key: "commentsCount",
-      render: (_, _asset) => <CustomTag value={0} />,
+      render: (_, asset) => {
+        const commentCount = asset.comments?.length || 0;
+        return <CustomTag value={commentCount} />;
```
Comment (KaWaite) [inline web/src/components/molecules/Asset/AssetListTable/index.tsx:61]: To keep this simple and concise just do 
```
<CustomTag value={asset.comments?.length || 0} />
```

File: web/src/components/organisms/Asset/convertAsset.ts [hunk 2]
```diff
@@ -13,5 +18,16 @@ export const convertAsset = (GQLAsset: GQLAsset | undefined): Asset | undefined
     projectId: GQLAsset.projectId,
     size: GQLAsset.size,
     url: GQLAsset.url,
+    threadId: GQLAsset.thread?.id ?? "",
+    comments: GQLAsset.thread?.comments?.map(comment => convertComment(comment)) ?? [],
+  };
+};
+
+export const convertComment = (GQLComment: GQLComment): Comment => {
```
Comment (KaWaite) [inline web/src/components/organisms/Asset/convertAsset.ts:26]: For now its fine, but notice we have I think a few convert functions now for converting to/from GQL types. 
It'd be nice to consolidate them in a single file (like we do in reearth-web).

Comment (nourbalaha) [reply web/src/components/organisms/Asset/convertAsset.ts:26]: Ok, this makes sense. I will do it in a separate PR.

File: web/src/components/molecules/Asset/AssetListTable/index.tsx (hunk 1 above)
Comment (nourbalaha) [reply web/src/components/molecules/Asset/AssetListTable/index.tsx]: Done!

**Review by KaWaite — APPROVED** [review]

**Pull request merged**